tests/TestingAppHost1/TestingAppHost1.AppHost/Program.cs

```diff
@@ -1,10 +1,16 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 
 var builder = DistributedApplication.CreateBuilder(args);
 
+builder.Services.ConfigureHttpClientDefaults(http =>
+{
+    http.AddStandardResilienceHandler();
+});
+
```

Comment:
What HTTP requests is the AppHost making?

Reply:
Not the AppHost itself, but the tests which are creating the `HttpClient` using that - see https://github.com/dotnet/aspire/pull/4500#discussion_r1638986530

Reply:
Can we just add this in the tests then? That way the "real app" isn't affected?

Reply:
Yeah, I'll try that. I wasn't sure what the idiomatic way of doing it was.

Reply:
In a "normal" test, you would do it here:

```csharp
    public async Task GetWebResourceRootReturnsOkStatusCode()
    {
        var appHost = await DistributedApplicationTestingBuilder.CreateAsync<Projects.AspireApp17_AppHost>();
        // HERE
        await using var app = await appHost.BuildAsync();
        await app.StartAsync();
```